{"@t":"2020-09-17T22:02:03.3537360Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-17T22:07:07.3719123Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":27,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-17T22:12:11.6041869Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":27,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-17T22:17:15.6257227Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":155,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-17T22:22:19.6458934Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":27,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-17T22:27:23.8724807Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":155,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-17T22:32:27.8855565Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":155,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-17T22:37:31.9078989Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":27,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-17T22:42:35.9150166Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":155,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-17T22:47:39.9347853Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":27,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-17T22:52:43.9461726Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":27,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-17T22:57:47.9663417Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":155,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-17T23:02:52.3321159Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":155,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-17T23:07:56.3534455Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":155,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-17T23:13:00.3713305Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":27,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-17T23:18:04.3932822Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":27,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-17T23:23:08.4165982Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":143,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-17T23:28:12.4372875Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":155,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-17T23:33:16.4599166Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":155,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-17T23:38:20.4761179Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-17T23:43:24.4968118Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":155,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-17T23:48:28.5175295Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":155,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-17T23:53:32.5381845Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":155,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-17T23:58:36.5578800Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":155,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T00:03:40.5806041Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T00:08:44.6012903Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T00:13:48.6181111Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T00:18:52.6385097Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":155,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T00:23:56.6471951Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T00:29:00.6649377Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T00:34:04.6836373Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T00:39:08.6933089Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T00:44:12.7090419Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T00:49:16.7267082Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T00:54:20.7464390Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T00:59:24.7711092Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T01:04:28.7947923Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T01:09:32.8045172Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T01:14:36.8202187Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T01:19:40.8369192Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T01:24:44.8566043Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T01:29:48.8653051Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T01:34:52.8830324Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T01:39:56.9027217Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T01:45:00.9124314Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T01:50:04.9301294Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T01:55:08.9498261Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T02:00:12.9744673Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T02:05:16.9931621Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T02:10:21.0108947Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T02:15:25.0325450Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T02:20:29.0522679Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T02:25:33.0729351Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T02:30:37.0946434Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T02:35:41.1143373Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T02:40:45.1360226Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T02:45:49.1547234Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T02:50:53.1634461Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T02:55:57.1830983Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T03:01:01.2068111Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T03:06:05.2244786Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T03:11:09.2461632Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T03:16:13.2678654Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T03:21:17.2885336Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T03:26:21.3052741Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T03:31:25.3229703Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T03:36:29.3456553Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T03:41:33.3653425Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T03:46:37.3820096Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T03:51:41.4007365Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T03:56:45.4194307Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T04:01:49.4311488Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T04:06:53.4537930Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T04:11:57.4625144Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T04:17:01.4832045Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T04:22:05.4919234Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T04:27:09.5086661Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T04:32:13.5283146Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-18T15:07:22.3154845Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":5,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
